app/components/error-wrapper.js

```diff
@@ -68,7 +68,7 @@ export default Ember.Component.extend({
     Updates the background based on the error class.
 
     @property observeErrorClass
-    @type error-wrapper.updateBackground
+    @type Ember.observer
```

Comment:
@rileytaylor it does make sense to document it as a method. See: http://yui.github.io/yuidoc/syntax/#method

I would suggest rewriting this as a `@method`